python/taichi/lang/__init__.py

```diff
@@ -473,7 +472,7 @@ def check_version():
     try:
         payload = json.dumps(payload)
         payload = payload.encode()
-        req = request.Request('http://metadata.taichi.graphics/check_version',
+        req = request.Request('https://metadata.taichi.graphics/check_version',
```

Comment:
In the future we might want to change this without changing source code ;)

Reply:
Good point. Fetching from vault might be a solution in the future.